versioned_docs/version-0.10/enterprise/install/releases.mdx

```diff
@@ -5,6 +5,30 @@ sidebar_label: Release Notes
 id: releases
 ---
 
+## 0.12.0
+DATE TBD
+
+### Improvements
+- Use `ingressClassName` field for ingresses instead of the deprecated `kubernetes.io/ingress.class` annotation
+- Okteto daemonset no longer mounts host volumes nor runs in privilege mode
+- Improve pipeline jobs lifecycle and properly handle timeouts. Logs for pipelines that time out are no longer lost
+- Invalid ingresses not accepted by the nginx ingress controller are now caught by the webhook
+- Ingresses are put to sleep alongside all other namespace resources and automatically wake by with incoming requests
+- Upgrade to Okteto CLI 2.2.1
```

Comment:
let's group all dependencie upgrades and put them at the bottom of the improvement list?